docs/debug_mode.md

```
@@ -0,0 +1,62 @@
+# Local Development
+
+  ## Debug Mode
```

markdownlint-cli2 (0.22.0)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)